react/components/BackToTopButton/styles.css

```
@@ -0,0 +1,21 @@
+.BackToTopButtonContainer {
+  border-bottom: 50px;
```

We usually prefer using `rem` instead of pixels, so if you could change this
```suggestion
  border-bottom: 3.125rem;
```
We use `font-size: 16px` that's why this value